fabfile.py

```diff
@@ -846,7 +846,7 @@ def install_builddeps():
         # may require a sudo
         if not run('brew link libevent', quiet=True):
             sudo('brew link libevent')
-        run('brew install zeromq libtool libmemcached gawk')
+        run('brew install zeromq libtool libmemcached gawk libxmlsec1')
```

Comment:
Quee? Used by python social auth to speed up the XML parsing of the SAML metadata? I'm confused as to how this made it in here...